_posts/2019-02-14-customize.md

```diff
@@ -280,18 +280,21 @@ Note: If you have created you own front-end or are using a [third-party plug-in]
 
 ## Change the default presentation
 
-When a new meeting starts, BigBlueButton displays a default presentation.  The file for the default presentation is located in `/var/www/bigbluebutton-default/default.pdf`.  You can replace the contents of this file with your presentation.  Whenever a meeting is created, BigBlueButton will automatically load, convert, and display this presentation for all users.
+When a new meeting starts, BigBlueButton displays a default presentation.  The file for the default presentation is located in `/var/www/bigbluebutton-default/default.pdf`.  You can replace the contents of this file with your presentation.  Whenever a meeting is created, BigBlueButton will automatically load, convert, and display this presentation for all users. On a bigbluebutton update this default may become replaced .
```

Comment:
There is one space too much at the end of the sentence. 

> On a bigbluebutton update this default may become replaced .

This is true for most changes which are made in the config, thus I don't think we should mention this a hundred times in the documentation. Also, sentence grammar.